vendor/github.com/ONSdigital/go-ns/zebedee/client/client.go

```diff
@@ -119,19 +120,16 @@ func (c *ZebedeeClient) GetDatasetLandingPage(path string) (data.DatasetLandingP
 	return dlp, nil
 }
 
-// SetAccessToken adds an access token to the client to authenticate with zebedee
-func (c *ZebedeeClient) SetAccessToken(token string) {
-	c.accessToken = token
-}
-
-func (c *ZebedeeClient) get(path string) ([]byte, error) {
+func (c *ZebedeeClient) get(ctx context.Context, path string) ([]byte, error) {
+	fmt.Println("URL >>>>>", c.zebedeeURL+path)
```

Comment:
Is this needed? I'm guessing this was here for debugging but if the output is useful we should use the logger.